.github/workflows/ci.yml

```
@@ -31,11 +33,11 @@ jobs:
           uv pip install --system pytest pytest-cov
           pytest --cov=src --cov-report=xml --cov-fail-under=70
       - name: Upload coverage
-        if: ${{ secrets.CODECOV_TOKEN != '' }}
+        if: env.CODECOV_TOKEN != ''
         uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)